test/regression/2015-08-05-symbolic-const-variable.c

```diff
@@ -0,0 +1,17 @@
+// RUN: %llvmgcc %s -emit-llvm -g -O0 -c -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: not %klee --output-dir=%t.klee-out -exit-on-error %t.bc 2> %t.log
```

Comment:
You should probably be checking for a particular error message here. Otherwise KLEE crashing or exiting for other reasons will make this test pass.